CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
+# v0.0.9
+
+- Updated return response for upload(file, signedDetails,callback,chunkSize,concurrency) API
```

Comment:
Breaking change: On success, response.data will now return an instance UploadResponse instead String.

Reply:
Done. Please review and resolve.